pydarn/plotting/fan.py

```diff
@@ -238,9 +214,115 @@ def plot_fan(cls, dmap_data: List[dict], ax=None, scan_index: int = 1,
 
-        return beam_corners_aacgm_lats, beam_corners_aacgm_lons, scan, grndsct, dtime
+        return beam_corners_aacgm_lats, beam_corners_aacgm_lons, scan, grndsct
+
+    @classmethod
+    def plot_fov(cls, stid: str, dtime: dt.datetime, ax=None,
+                 lowlat: int = 50, ranges: List = [0, 75],
+                 boundary: bool = True, fov_color: str = None,
+                 alpha: int = 0.5):
+        """
+        plots only the field of view (FOV) for a given radar station ID (stid)
+
+        Parameters
+        -----------
+            stid: str
+                Radar station ID
+            ax: matplotlib.pyplot axis
+                Pre-defined axis object to pass in, must currently be
+                polar projection
+                Default: Generates a polar projection for the user
+                with MLT/latitude labels
+            dtime: datetime datetime object
+                sets the datetime used to find the coordinates of the
+                FOV
+            lowlat: int
+                Lower AACGM latitude boundary for the polar plot
+                Default: 50
+            ranges: list
+                Set to a two element list of the lower and upper ranges to plot
+                Default: [0,75]
+            boundary: bool
+                Set to false to not plot the outline of the FOV
+                Default: True
+            fov_color: str
+                fov color to fill in the boundary
+                default: None
+            alpha: int
+                alpha controls the transparency of
+                the fov color
+                Default: 0.5
+
+        Returns
+        -------
+            beam_corners_aacgm_lats - list of beam corners AACGM latitudes
+            beam_corners_aacgm_lons - list of beam corners AACGM longitudes
+            thetas - theta polar coordinates
+            rs - radius polar coordinates
+        """
+        # Get radar beam/gate locations
+        beam_corners_aacgm_lats, beam_corners_aacgm_lons = \
+            radar_fov(stid, coords='aacgm', date=dtime)
+        fan_shape = beam_corners_aacgm_lons.shape
+
+        # Work out shift due in MLT
+        beam_corners_mlts = np.zeros((fan_shape[0], fan_shape[1]))
+        mltshift = beam_corners_aacgm_lons[0, 0] - \
+            (aacgmv2.convert_mlt(beam_corners_aacgm_lons[0, 0], dtime) * 15)
+        beam_corners_mlts = beam_corners_aacgm_lons - mltshift
+
+        # Hold the beam positions
+        thetas = np.radians(beam_corners_mlts)
+        rs = beam_corners_aacgm_lats
+
+        # Setup plot
+        # This may screw up references
+        if ax is None:
+            ax = plt.axes(polar=True)
+            if beam_corners_aacgm_lats[0, 0] > 0:
+                ax.set_ylim(90, lowlat)
+                ax.set_yticks(np.arange(lowlat, 90, 10))
+            else:
+                ax.set_ylim(-90, -lowlat)
+                ax.set_yticks(np.arange(-lowlat, -90, -10))
```

Comment:
```suggestion
                ax.set_ylim(-90, -abs(lowlat))
                ax.set_yticks(np.arange(-abs(lowlat), -90, -10))
```
Same changes in fan plotting review